source/draw/gpu/opengl/OpenGLCanvas.ooc

```diff
@@ -54,7 +54,7 @@ OpenGLCanvas: class extends OpenGLSurface {
 					gpuMap add("texture0", image)
 			}
 		}
-		gpuMap use(this _target)
+		gpuMap use(this _target, projection * view * model, textureTransform)
```

Comment:
Doing the multiplication here makes it easier to refactor model-view-projection.